### ISSUE TITLE
A little guidance with crew voices

### ISSUE THREAD

**Issue by Picroc** [GaijinEntertainment/fmod_studio_warthunder_for_modders#123]
Sorry if I'm being too stupid, but poking around the Fmod Studio and searching elsewhere for info didn't help.

What is the best way to replace various voicelines for the crew with this different languages system? As I understand, Programmer Instruments are used to then map (using these instruments' names) events to specific language files. So, editing events in any way is not possible?

I even tried replacing the audio files in assets folder (with keeping the old file name) and building both banks with events and with audio table for my language, with the result still hearing the original voice lines. Plus, this way is really unpleasant and I assume wrong.

Would be thankful for any advice, 'cause I'm out of options now.

**Comment by Uchuujin01** [comment]
Hello! 

> I even tried replacing the audio files in assets folder (with keeping the old file name) and building both banks with events and with audio table for my language, with the result still hearing the original voice lines. Plus, this way is really unpleasant and I assume wrong.

This is exactly how fmod localisation tables work, so, no, it's not wrong. You have the same file names for every language, and you trigger them with Programmer Instrument, that contains name (including relative subdirectories).

So, technically, I can't see what you did wrong, try to check audiotables again. But, this is strange you hear the old voicelines, did you switched on sound mod mode in config.blk?

**Comment by Picroc** [comment]
@Uchuujin01 Yeah, I've checked it with another sound mod and it worked. I guess I'll try replacing the files again
